rolling/distribution.yaml

```diff
@@ -849,6 +849,11 @@ repositories:
       type: git
       url: https://github.com/dynamixel-community/dynamixel_hardware.git
       version: rolling
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/dynamixel-community/dynamixel_hardware-release.git
```

Comment:
Our best practices here are to use an https://github.com/ros2-gbp repository for releases into Rolling.  That way, we'll be able to automatically release this package from Rolling into the next ROS distribution (Iron).  If you'd like to do this, then please see https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md on how to create one.

Reply:
I have opened https://github.com/ros2-gbp/ros2-gbp-github-org/pull/160 to move the release repository.

Reply:
Thank you very much for your help. I'd like to transfer the repository.

Reply:
I've mirrored the current repository and the permissions updates are deploying now.